### ISSUE TITLE
Information on contact page doesnt link to anything which is a common convention 

### ISSUE THREAD

**Issue by 16OscarClarke** [16OscarClarke/2.4_2.8DITWebsiteInternal#9]
all contact details on the contact page cannot interacted with
